### ISSUE TITLE
Reserved Peers not priorized ?!

### ISSUE THREAD

**Issue by SurfingNerd** [DMDcoin/diamond-node#149]
Reserved Peers was understood that the system "always" tries to connect with them.
observations on the beta network has shown that this might be wrong.

The Node should even kick out other Nodes to connect to the reserved peers, and retry if the session is lost.

If that is not possible, we should introduce an "Enforced Peers" Feature that really makes sure that the connection can be established.

**Comment by diamond-info** [comment]
Why not implement several v6 reference IP addresses that will stabilize the network? IPV6 nodes are much more protected from classic DDoS attacks to which IP v4 addresses are exposed.

**Comment by cryptonit** [comment]
if we support ipv6 then it would be amandatory requoirement for all validators to be able interact with ipv6 destinations
there still internet providers and datacenters without ipv6 support

**Comment by diamond-info** [comment]
Obviously, IPv6 is the future (it can provide up to 5*10 to the 28th degree of addresses) since IPv4 in free form is almost over.
